repeat_expansion_variants/pipeline.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+"""A pipeline to extract repeat expansion variants from ClinVar TSV dump. For documentation refer to README.md"""
+
+import gzip
+from io import StringIO
+import logging
+
+import numpy as np
+import pandas as pd
+
+from . import biomart, clinvar_identifier_parsing
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def none_to_nan(*args):
+    """Converts all arguments which are None to np.nan, for consistency inside a Pandas dataframe."""
+    return [np.nan if a is None else a for a in args]
+
+
+def load_clinvar_data(clinvar_summary_tsv):
+    """Load ClinVar data, preprocess, and return it as a Pandas dataframe."""
+    # Load and pre-filter the file
+    gzip_stream = gzip.open(clinvar_summary_tsv, 'r')
+    repeat_expansion_data = ''.join([
+        line.decode() for i, line in enumerate(gzip_stream)
+        if i == 0 or line.decode().split('\t')[1] == 'NT expansion'
+    ])
+    # Load variants
+    variants = pd.read_table(StringIO(repeat_expansion_data), low_memory=False)
+    # Filter only NT expansion variants
+    variants = variants[variants['Type'] == 'NT expansion']
```

Comment:
Seems to have been done in line 29 already...

Reply:
You're right! Fixed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/3/commits/45e54e50507450b2530ee10145ca04e5d01d9426